src/DIRAC/ProductionSystem/DB/ProductionDB.sql

```diff
@@ -25,7 +25,8 @@ CREATE TABLE Productions(
     Description LONGTEXT,
     CreationDate DATETIME,
     LastUpdate DATETIME,
-    AuthorDN VARCHAR(255) NOT NULL,
+    Author VARCHAR(255) NOT NULL,
+    AuthorDN VARCHAR(255),
```

Comment:
```suggestion
    AuthorDN VARCHAR(255) DEFAULT '',
```
Wasn't there a problem with the transformationDB about no default value for AuthorDN?

Reply:
Does not seem so: https://github.com/DIRACGrid/DIRAC/pull/7124/files#diff-c9f689c0427d70b7ff70c68598d409ab7d16f597e4fe484e24627ab893bda479

Reply:
In the hackathon on August 3rd there was a problem about AuthorDN not having a default value, wasn't there?

Reply:
I don't remember about AuthorDN issues for TrasformationSystem, but there was a similar thing for WMS, for which I created https://github.com/DIRACGrid/DIRAC/pull/7155/files

Reply:
On the certif DB for TransformationDB.Transformations: 
```
`AuthorDN` varchar(255) DEFAULT NULL,`
```